.github/workflows/turing.yaml

```diff
@@ -211,10 +209,10 @@ jobs:
       - name: Check out code
         uses: actions/checkout@v2
 
-      - name: Set up Go 1.14
+      - name: Set up Go 1.18
         uses: actions/setup-go@v2
         with:
-          go-version: "1.14"
+          go-version: "1.18"
```

Comment:
Curious, is it possible to use env var to control go-version everywhere?

Reply:
Let me change it to use an envvar instead.

Reply:
Additionally i changed it for the Go linter as well.